lib/foreman_docker/engine.rb

```diff
@@ -62,6 +62,7 @@ class Engine < ::Rails::Engine
           permission :create_containers,  :'containers/steps' => [:show, :update],
                                           :containers         => [:new]
           permission :destroy_containers, :containers         => [:destroy]
+          permission :power_compute_resources_vms, :containers => [:power]
```

Comment:
I wanted to keep the original permission instead of introducing docker-container-specific one (they are the same after all).